bindings/wasm/examples/vite.config.js

```diff
@@ -7,6 +7,18 @@ export default defineConfig({
   worker: {
     format: 'es',
   },
+  plugins: [
+    {
+      name: 'configure-response-headers',
+      configureServer: (server) => {
+        server.middlewares.use((_req, res, next) => {
+          res.setHeader('Cross-Origin-Embedder-Policy', 'require-corp');
```

Comment:
This is the cross-origin policy required for SharedArrayBuffer to work, there is no way around this. Hope that github pages supports this.